uni/lib/view/bus_stop_next_arrivals/bus_stop_next_arrivals.dart

```diff
@@ -78,14 +79,33 @@ class NextArrivalsState extends State<NextArrivals> {
   /// Returns a list of widgets for a successfull request
   List<Widget> requestSuccessful(context) {
     final List<Widget> result = <Widget>[];
+    final List<String> images = ['assets/images/bus.png', 'assets/images/flat_bus.png'];
 
     result.addAll(getHeader(context));
 
     if (widget.buses.isNotEmpty) {
       result.addAll(getContent(context));
     } else {
-      result.add(Text('Não existe nenhuma paragem configurada',
-          style: Theme.of(context).textTheme.headline6));
+      result.add(
```

Comment:
Ideally, instead (or on top of) `RotatingImage` you would have a generic widget that handles the empty/not empty cases by itself.